src/components/utils/jwt.ts

```diff
@@ -1,28 +1,34 @@
 import { currentSeconds } from './index'
+import type { UrlsConfig } from '~types/commons'
+import type { EnterpriseFeatures } from '~types/enterprise'
 
-export const parseJwt = (token) => {
+export const parseJwt = (token: string): Record<string, unknown> => {
```

Comment:
` Record<string, unknown>` Does this mean that the function will return an object with a string as a key and an unknown value?

Reply:
[Yes, exactly](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype)